### PR TITLE
View validation compatibility with Studio.

### PR DESCRIPTION
**Pull request by kujiu**
Add attributes and elements on XML views' checks to avoid errors with Studio (renamed_field and studio_groups)

### CODE REVIEW

File: odoo/addons/base/rng/common.rng [hunk 1]
```diff
@@ -270,6 +275,89 @@
...
+                    <rng:ref name="filter"/>
+                    <rng:ref name="html"/>
+                    <rng:element name="newline"><rng:empty/></rng:element>
+                </rng:choice>
+            </rng:zeroOrMore>
+        </rng:element>
+    </rng:define>
+
+    <rng:define name="renamed_field">
+        <rng:element name="field">
```
Comment (rco-odoo) [inline odoo/addons/base/rng/common.rng]: name should be `renamed_field`

Comment (rco-odoo) [reply odoo/addons/base/rng/common.rng]: redefine `field` and `renamed_field` as
```xml
<rng:define name="field">
    <rng:element name="field">
        <rng:ref name="fieldContent"/>
    </rng:element>
</rng:define>

<rng:define name="renamed_field">
    <rng:element name="renamed_field">
        <rng:ref name="fieldContent"/>
    </rng:element>
</rng:define>

<rng:define name="fieldContent">
    <!-- put all the common content here -->
</rng:define>
```

**Comment by KangOl** [comment]
Why is it part of core?
Can't it be overwritten in `web_studio` module?

**Comment by KangOl** [comment]
And it's break runbot: http://runbot.odoo.com/runbot/build/266342

**Pull request merged**